hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This release fixes a very rare edge case in the test-case mutator.
+There is no user-visible change.
```

Comment:
I don't think it's accurate to say that there is no user visible change. Users can definitely hit this bug in theory!